tests-e2e/ocw-ci-test-course/nav-links.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { test, expect } from "@playwright/test"
+import { CoursePage } from "../util"
+
+test("Navbar links working properly", async ({ page }) => {
+  const course = new CoursePage(page, "course")
+  await course.goto("/")
+
+  const navbar = await page
+    .locator("#course-main-content div")
+    .filter({
+      hasText:
+        "Section 1 Menu Title Subsection 1a Menu Title Subsection 1b Menu Title Section 2",
+    })
+    .nth(3)
+  const linkCount = await navbar.locator("a").count()
+  for (let i = 0; i < linkCount; i++) {
+    const link = await navbar.locator("a").nth(i)
+    const url = await link.getAttribute("href")
+    await link.click()
+    await expect(page.url()).toContain(url)
```

Comment:
Is it possible to check here if the page returned a 404?

Or maybe check if all URLs return a 200?

I'm not sure. Not seeing a 404 means that the link was correct, so I think that check should be enough.

Reply:
let me check, I think, if 404 page renders some different div with specific ID or class we can grab, we can expect that not to be visible?